control-plane/gateways/serviceaccount_test.go

```diff
@@ -16,6 +17,13 @@ func TestNewMeshGatewayBuilder_ServiceAccount(t *testing.T) {
 			Namespace: "default",
 			Name:      "mesh-gateway",
 		},
+	}, GatewayConfig{}, &meshv2beta1.GatewayClassConfig{
+		TypeMeta: metav1.TypeMeta{},
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "test-gcc",
+		},
+		Spec:   pbmesh.GatewayClassConfig{},
```

Comment:
This type actually _just_ changed in `main` due to #3297 and will instead be `meshv2beta1.GatewayClassConfigSpec` on rebase